Gemfile

```diff
@@ -26,6 +26,8 @@ gem 'bootsnap', '>= 1.4.2', require: false
 # gem 'rack-cors'
 
 group :development, :test do
+  # help to kill N+1 queries and unused eager loading
```

Comment:
Do we want to keep comments on gems or not? (I like both approachs as long as we are consistent)

Reply:
mmm I agree with you regarding to keep consistency. I prefer to go with the approach of no comments on gems.